web/src/features/moreCast2/components/GridComponentRenderer.tsx

```diff
@@ -36,19 +39,52 @@ export class GridComponentRenderer {
   ) => {
     return <ForecastHeader colDef={params.colDef} columnClickHandlerProps={columnClickHandlerProps} />
   }
-  public renderHeaderWith = (params: GridColumnHeaderParams) => {
+  public renderHeaderWith = (
+    params: Pick<GridColumnHeaderParams, 'field'> & {
+      colDef: Pick<GridColumnHeaderParams['colDef'], 'field' | 'headerName'>
+    },
+    allRows?: MoreCast2Row[]
+  ) => {
+    const headerName = params.colDef.headerName ?? ''
+
     if (params.field.endsWith('_BIAS')) {
-      const headerName = params.colDef.headerName ?? ''
       const index = headerName.indexOf('_BIAS')
       const prefix = headerName.slice(0, index)
       return (
         <div data-testid={`${params.colDef.field}-column-header`}>
-          <Box sx={{ height: '1rem' }}>{prefix}</Box>
-          <Box>bias</Box>
+          <Typography data-testid={`${params.colDef.field}-column-header`} style={{ fontSize: '14px' }}>
+            {prefix}
+          </Typography>
+          <Typography style={{ fontSize: '14px' }}>bias</Typography>
+        </div>
+      )
+    }
+    const modelType = weatherModelsWithTooltips.find(model => headerName === model)
+    if (modelType && allRows && allRows.length > 0) {
+      const timestampField = `predictionRunTimestamp${modelType}` as keyof MoreCast2Row
+      const timestamp = allRows[0][timestampField] as string | null | undefined
+      return (
+        <div style={{ display: 'flex', alignItems: 'center' }}>
+          <Typography style={{ fontSize: '14px' }}>{headerName}</Typography>
+          {timestamp && (
+            <Tooltip
+              data-testid={`${params.colDef.field}-model-run-tooltip`}
+              title={`Model run: ${DateTime.fromISO(timestamp).toFormat('MMM dd, yyyy HH:mm ZZZZ')}`}
```

Comment:
We could check in with the forecasters but when they talk about model runs I think they refer to the UTC times. It might slightly reduce cognitive load if the displayed date doesn't have to be mentally converted from PDT to UTC.